<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParaWindow</class>
 <widget class="QMainWindow" name="ParaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="update_pushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>651</width>
      <height>201</height>
     </rect>
    </property>
    <property name="title">
     <string>测距参数</string>
    </property>
    <widget class="QLineEdit" name="errorRange_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="errorRange_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头常数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara0_Label_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>偏差范围：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara0_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头一次项系数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara1_Label_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara1_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头二次项系数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara2_Label_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara2_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>摄像头三次项系数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara3_Label_1">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara3_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);</string>
     </property>
     <property name="text">
      <string>摄像头1参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara1_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>110</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara3_Label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara3_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>170</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara1_Label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara2_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>140</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara0_Label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);</string>
     </property>
     <property name="text">
      <string>摄像头2参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="cameraPara0_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>80</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="cameraPara2_Label_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>140</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>匹配参数</string>
    </property>
    <widget class="QLineEdit" name="minRatio_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="matchType_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="minHessian_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QComboBox" name="matchType_comboBox">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>110</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ORB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SURF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SIFT</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>黑塞矩阵阈值/特征数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>匹配算法：</string>
     </property>
    </widget>
    <widget class="QLabel" name="minRatio_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="minHessian_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>筛选比例范围：</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancel_pushButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="default_pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>默认</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>YOLO参数</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>置信阈值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="confThreshold_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confThreshold_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>非最大抑制：</string>
     </property>
    </widget>
    <widget class="QLabel" name="nmsThreshold_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nmsThreshold_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图像网络高度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>图像网络宽度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="inpWidth_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inpWidth_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inpHeight_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>110</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="inpHeight_Label">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>251</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>定位参数</string>
    </property>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);</string>
     </property>
     <property name="text">
      <string>摄像头1参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="k_Para_Label_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="b_Para_Label_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="k_Para_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="b_Para_lineEdit_1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>k:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(229, 229, 229);</string>
     </property>
     <property name="text">
      <string>摄像头2参数</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="k_Para_Label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>b:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="b_Para_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="b_Para_Label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(173, 173, 173);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>k:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="k_Para_lineEdit_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>170</y>
       <width>110</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
